### PR TITLE
add instagram post ui

### PR DESCRIPTION
**Pull request by Mo30-you**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,15 +12,22 @@
...
     "@remix-run/serve": "^1.14.3",
     "isbot": "^3.6.5",
+    "pnpm": "^8.1.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@remix-run/dev": "^1.14.3",
     "@remix-run/eslint-config": "^1.14.3",
+    "@svgr/cli": "^7.0.0",
```
Comment (ali-shabani) [inline package.json:22]: why you added unnecassary packages?
please remove them

Comment (ali-shabani) [reply package.json:22]: @Mo30-you

**Review by ali-shabani — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "@remix-run/react": "^1.14.3",
     "@remix-run/serve": "^1.14.3",
     "isbot": "^3.6.5",
+    "pnpm": "^8.1.0",
```
Comment (ali-shabani) [inline package.json:15]: remove `pnpm` dependency

**Review by ali-shabani — CHANGES_REQUESTED** [review]

**Comment by Mo30-you** [comment]
do it changeing

**Comment by Mo30-you** [comment]
delete pnpm

**Comment by Mo30-you** [comment]
delete packege lock

**Pull request merged**